src/walletConnect/v2/saga.ts

```diff
@@ -419,6 +419,7 @@ function* handleAcceptRequest({ request }: AcceptRequest) {
       ...defaultTrackedProperties,
       error: e.message,
     })
+    yield put(denyRequest(request, getSdkError('USER_REJECTED')))
```

Comment:
there could be many errors, i am not sure which one we should send back to the WC server but USER_REJECTED seems accurate enough?

Reply:
Similar here, but I think we should also directly reject too.